### PR TITLE
Bug 945507 - Pick up marionette-js-runner@1.0 a=test-only r=Eli,lightsofapollo

### PR DESCRIPTION
**Pull request by lambdabaa**

### CODE REVIEW

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[gaye Gareth Aye (gaye)](https://github.com/gaye) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=bd435e063867)

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[gaye Gareth Aye (gaye)](https://github.com/gaye) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=4bd29107784c)

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[gaye Gareth Aye (gaye)](https://github.com/gaye) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=44ffc96990d9)

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[gaye Gareth Aye (gaye)](https://github.com/gaye) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=5a5ac3e3e4f6)

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[gaye Gareth Aye (gaye)](https://github.com/gaye) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=d0e183701744)

File: tests/performance/startup_events_test.js [hunk 1]
```diff
@@ -67,7 +67,8 @@ marionette('startup event test > ' + appPath + ' >', function() {
 
         app.close();
 
-        assert.ok(memoryUsage, 'could not collect memory usage');
+        // TODO: Should be able to fetch memory usage.
+        //assert.ok(memoryUsage, 'could not collect memory usage');
```
Comment (lightsofapollo) [inline tests/performance/startup_events_test.js]: pretty sure this should be process.memoryUsage anyway

**Comment by try-server-hook** [comment]
[gaye Gareth Aye (gaye)](https://github.com/gaye) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=9549ffa0234f)

**Comment by try-server-hook** [comment]
[gaye Gareth Aye (gaye)](https://github.com/gaye) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=4fcacb6a3e4f)

**Comment by try-server-hook** [comment]
[gaye Gareth Aye (gaye)](https://github.com/gaye) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=f258374223da)

**Comment by try-server-hook** [comment]
[gaye Gareth Aye (gaye)](https://github.com/gaye) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=17c6d364fb10)

**Comment by try-server-hook** [comment]
[gaye Gareth Aye (gaye)](https://github.com/gaye) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=5ad445589ae5)

**Comment by try-server-hook** [comment]
[gaye Gareth Aye (gaye)](https://github.com/gaye) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=be0077c7e796)

**Comment by try-server-hook** [comment]
[gaye Gareth Aye (gaye)](https://github.com/gaye) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=0e39958be081)

**Comment by try-server-hook** [comment]
[gaye Gareth Aye (gaye)](https://github.com/gaye) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=8217b2ad011e)

**Comment by try-server-hook** [comment]
[gaye Gareth Aye (gaye)](https://github.com/gaye) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=8a2e2aa5327b)

**Comment by try-server-hook** [comment]
[gaye Gareth Aye (gaye)](https://github.com/gaye) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=63a9fa5b4b4b)

**Comment by try-server-hook** [comment]
[gaye Gareth Aye (gaye)](https://github.com/gaye) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=efa51f0ec308)

**Comment by try-server-hook** [comment]
[gaye Gareth Aye (gaye)](https://github.com/gaye) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=098543ced559)

**Comment by try-server-hook** [comment]
[gaye Gareth Aye (gaye)](https://github.com/gaye) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=ec3e6c3368c6)

**Pull request closed without merging**